Converters/ArcGIS/Speckle.Converters.ArcGIS3/ToSpeckle/Helpers/PropertiesExtractor.cs

```diff
@@ -0,0 +1,54 @@
+namespace Speckle.Converters.ArcGIS3.ToSpeckle.Helpers;
+
+public sealed class PropertiesExtractor
+{
+  public PropertiesExtractor() { }
+
+  public Dictionary<string, object?> GetProperties(AC.CoreObjectsBase coreObjectsBase)
+  {
+    switch (coreObjectsBase)
+    {
+      case ACD.Row row:
+        return GetRowFields(row);
+    }
+
+    return new();
+  }
+
+  public Dictionary<string, object?> GetRowFields(ACD.Row row)
+  {
+    Dictionary<string, object?> rowFields = new();
+    foreach (ACD.Field field in row.GetFields())
+    {
+      // POC: do not set null values
+      // POC: we are not filtering by the layer visible fields
+      if (FieldValueToSpeckle(row, field) is object value)
```

Comment:
even if it returns null, i guess it is still `object`? below statement might be better
```c#
var value = FieldValueToSpeckle(row, field);
if (value != null)
{
    rowFields[field.Name] = value;
}
```